### ISSUE TITLE
Narrator doesn't announce selection state of textbox selected text in modal NOR deletion status on typing text field

### ISSUE THREAD

**Issue by James-Matthews** [microsoft/microsoft-ui-xaml#1951]
**Describe the bug**
If a textbox in a modal popup has the text selected by default, the narrator does not read out the selected status of the text, nor does it announce deletion status once you do start typing

**Steps to reproduce the bug**

1. Create a blank UWP with a ContentDialog that opens on clicking a button
2. Add a TextBox to that ContentDialog with text already set
3. Ensure focus of text box TB.GotFocus += (p1, p2) => TB.SelectAll();
4. See that narrator does not read selected status or deletion of text

**Expected behavior**
Text to edit is selected by default and the narrator should announce as selected.
On start typing the text, the text in the edit field gets deleted. Hence the narrator should give an announcement.

**Version Info**

NuGet package version: 
2.3.191211002 and 2.2.190917002

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| November 2019 Update (18363)  | Yes  |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes   |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Additional context**
